apps/desktop/src/components/views/terminal/terminal-tab-button.tsx

```
@@ -0,0 +1,100 @@
+    setIsEditing(false);
+  };
+
+  return (
+    <div
+      onClick={onClick}
+      onDoubleClick={handleDoubleClick}
+      className={cn(
+        'group flex cursor-pointer items-center gap-2 border-r border-border px-4 py-2',
+        isActive && 'bg-background',
+      )}
+    >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)